### PR TITLE
[SOC] Translate dev_quick_start

### PR DESCRIPTION
**Pull request by QuakeWang**
According to [Dev_Quick_Start](https://dolphinscheduler.apache.org/zh-cn/docs/dev/user_doc/dev_quick_start.html) and translate it.

### CODE REVIEW

**Comment by CalvinKirs** [comment]
It is recommended to set the dev default document in site_config/site.js to dev_quick_start

**Comment by QuakeWang** [comment]
> It is recommended to set the dev default document in site_config/site.js to dev_quick_start

Do you mean to add the English version of dev_quick_start in site_config/site.js?

File: docs/zh-cn/dev/user_doc/dev_quick_start.md [hunk 1]
```diff
@@ -14,37 +14,34 @@ git clone git@github.com:apache/dolphinscheduler.git
 
 #### 2: 启动后端
 ```
-mvn install -DskipTests
+mvn -U install package -Prelease -Dmaven.test.skip=true
 ```
```
Comment (CalvinKirs) [inline docs/zh-cn/dev/user_doc/dev_quick_start.md:18]: Why do we need to add -Prelease？

**Comment by QuakeWang** [comment]
I have updated it.

**Comment by CalvinKirs** [comment]
@ptyp PTAL

File: docs/en-us/dev/user_doc/dev_quick_start.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+Note: Only for standalone development debugging use, the default use of H2 Database, Zookeeper Testing Server.
```
Comment (ptyp) [inline docs/en-us/dev/user_doc/dev_quick_start.md:1]: "The default option is H2 Database and Zookeeper Testing Server" would be more natural

File: docs/zh-cn/dev/user_doc/dev_quick_start.md (hunk 1 above)
Comment (CalvinKirs) [reply docs/zh-cn/dev/user_doc/dev_quick_start.md]: English version has not been modified

File: docs/en-us/dev/user_doc/dev_quick_start.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+ nvm install v12.20.2
+
+note: Mac users could install npm through brew: brew install npm
+
+###### d. Validate the node installation
+
+ node --version
+
+##### ii. cd dolphinscheduler-ui and run the following command:
```
Comment (ptyp) [inline docs/en-us/dev/user_doc/dev_quick_start.md:47]: command=》commands

**Review by ptyp — COMMENTED** [review]
Good job！

File: docs/en-us/dev/user_doc/dev_quick_start.md (hunk 2 above)
Comment (QuakeWang) [reply docs/en-us/dev/user_doc/dev_quick_start.md]: Thanks for reviewing, I have updated it.

**Review by ptyp — APPROVED** [review]

**Review by dailidong — APPROVED** [review]
+1
good job

**Pull request merged**